projects/self_feeding/modules.py

```diff
@@ -164,17 +164,7 @@ def score_similarity(self, context_h, cand_h):
                 'Unexpected candidate dimensions {}' ''.format(cand_h.dim())
             )
 
-        return self.normalize_scores(scores)
-
-    def normalize_scores(self, scores):
-        if self.opt['scores_norm'] == 'dot':
-            return scores
-        elif self.opt['scores_norm'] == 'sqrt':
-            return scores / math.sqrt(self.opt['embedding_size'])
-        elif self.opt['scores_norm'] == 'dim':
-            return scores / self.opt['embedding_size']
-        else:
-            raise ValueError
+        return scores
```

Comment:
@emilydinan can you confirm this please?

Reply:
this looks ok to me. we typically do this outside of the module so that we can pass pre-encoded candidates to the model.